version.py

```diff
@@ -0,0 +1,34 @@
+from os import getenv
+
+import git
+import textwrap
+
+
+def version():
+    ver = open('VERSION').read().strip()
+    repo = git.Repo('.')
+    head = repo.head.object
+    msg = head.message
+    commit = str(head)
+    build_number = getenv('CIRCLE_BUILD_NUM', '0')
+    return ver, msg, commit, build_number
+
+
+def version_string(ver, msg, commit, n):
+    return '{}.{}~{}'.format(ver, n, commit[:7])
+
+
+def write_changelog():
+    import debian.changelog
```

Comment:
What's the purpose of importing it here?

Reply:
@rptrchv To write debian/changelog. We only need it when building deb package, so we import it here to avoid adding extra dependencies when we don't need them.

Reply:
I see